mllib/src/test/scala/org/apache/spark/ml/feature/MaxAbsScalerSuite.scala

```diff
@@ -45,9 +44,9 @@ class MaxAbsScalerSuite extends SparkFunSuite with MLlibTestSparkContext with De
       .setOutputCol("scaled")
 
     val model = scaler.fit(df)
-    model.transform(df).select("expected", "scaled").collect()
-      .foreach { case Row(vector1: Vector, vector2: Vector) =>
-      assert(vector1.equals(vector2), s"MaxAbsScaler ut error: $vector2 should be $vector1")
+    testTransformer[(Vector, Vector)](df, model, "expected", "scaled") {
+      case Row(vector1: Vector, vector2: Vector) =>
+        assert(vector1.equals(vector2), s"MaxAbsScaler error: $vector2 should be $vector1")
```

Comment:
`vector1 === vector2`
and the error message looks strange. ==> s"scaled value $vector2 do not equal expected value "$vector1" ?